server.js

```diff
@@ -0,0 +1,41 @@
+var cluster = require('cluster');
+var path = require('path');
+var log = require('util').debuglog('master');
+
+if(cluster.isMaster) {
+	cluster.setupMaster();
+
+	log('Starting workers');
+	// start the api server
+	cluster.fork({
+		script: 'src/api/index.js',
+		port: process.env.npm_package_config_api_port
+	});
+	log('Started api server');
```

Comment:
It doesn't look like I can detect it from this side in any case. I do need to add a listener for workers dying, if only to log accurate messages about which workers are still running (and restart the ones that die).